users/migrations/0012_auto_20180327_1440.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.5 on 2018-03-27 12:40
+from __future__ import unicode_literals
```

Comment:
s.a.

Reply:
?

Reply:
Should I fix the django.po, or do you do that?